### DISCUSSION TITLE
Option to have the  "Ground Items" plugin also color the despawn timer

### DISCUSSION THREAD

**Discussion by Stickaplex** [runelite/runelite#18395] [Ideas]
With how you can customize the colors of dropped items either manually or depending on the items value, would be great if you could color the despawn timer next to it the same color. Potentially have options to have 2-3 colors per despawn timer. (just dropped, shows up for everyone, about to despawn)

Just would make it nicer visually I think

**Comment by scathecraw** [comment]
Agreed with this suggestion. I'd like to remove some visual clutter and having this able to be a custom color or progression of colors would be great, or a way to match it to certain value of drops so you can prioritize at a glance.
